datahub-web-react/src/app/home/HomePageHeader.tsx

```diff
@@ -232,11 +206,11 @@ export const HomePageHeader = () => {
                         autoCompleteStyle={styles.searchBox}
                         entityRegistry={entityRegistry}
                     />
-                    {suggestionsToShow && suggestionsToShow.length > 0 && (
+                    {searchResultsToShow && searchResultsToShow.length > 0 && (
```

Comment:
minor: we can simplify to 

`searchResultsToShow?.length > 0`

Reply:
As We are using 
 
`let result: string[] | undefined = [];`

That's why added check `searchResultsToShow && searchResultsToShow.length > 0`.